posthog/api/cohort.py

```diff
@@ -54,7 +54,8 @@ def _handle_csv(self, file, cohort: Cohort) -> None:
     def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Cohort:
         request = self.context["request"]
         validated_data["created_by"] = request.user
-        validated_data["is_calculating"] = True
+        if not request.FILES.get("csv"):
```

Comment:
Shouldn't the condition be the same as in #update, `if not validated_data.get("is_static")`? Otherwise I'd be wondering every time when reading this code why the asymmetry.

Reply:
Yeah makes sense, fixed